addons/hr/models/hr_departure_reason.py

```diff
@@ -12,17 +12,35 @@ class HrDepartureReason(models.Model):
 
     sequence = fields.Integer("Sequence", default=10)
     name = fields.Char(string="Reason", required=True, translate=True)
-    reason_code = fields.Integer()
```

Comment:
Small issue with this field removal.

Removing the code prevents creating a new one and assigning it behavior similar to an existing one without having to modify the Python code.

For example, 'Resigned: Doesn’t like management' will not give a correct calculation of the notice period if you cannot assign it the same code as the standard 'Resigned.'

Plus, it encourages making calls to ref instead of just checking the code, which consequently always impacts performance.

Reply:
he doesn't really remove the reason_code he wants to create a reason_Code per country; you think it's a bad idea ?

Reply:
Exactly :)
It's moved to `l10n_be` because it's the code for belgium: https://github.com/odoo/enterprise/blob/24da5ab871d16ae6ee4a7e5a8b3de4b69b008318/l10n_be_hr_payroll/models/hr_departure_reason.py#L9